src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write code here
+const sortList = (list) => {
+  [...list].map((li) => {
+    const salary = li
+      .getAttribute('data-salary')
+      .slice(1, -1)
+      .replace(',', '.');
+
+    li.setAttribute('salary', salary);
+  });
+
+  const sorted = [...list].sort((a, b) => {
+    const salaryA = a.getAttribute('salary');
+    const salaryB = b.getAttribute('salary');
+
+    return salaryB - salaryA;
```

Comment:
Ensure that the salary attributes are numbers before performing subtraction. If they are strings, the subtraction will not work as intended. Consider converting them to numbers using `parseFloat` or a similar method.